### DISCUSSION TITLE
"Apply palette if one exists" checkbox?

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2385] [Ideas]
I was thinking, could we have a checkbox to automatically apply a palette when selecting a given LXQt theme, if the user so wishes? It should be fairly simple logic: if the box is checked, the system will look for a file under $PATH/lxqt/palettes/ that matches the selected theme name and apply it if it exists, else do nothing.

**Comment by stefonarch** [comment]
We have this idea laying around somewhere already, but couldn't find it atm.
